src/EdgeGPT.py

```diff
@@ -201,21 +176,18 @@ async def ask_stream(self, prompt: str) -> str:
         self.request.update(prompt=prompt)
         # Send request
         await self.wss.send(append_identifier(self.request.struct))
-        stream_cache = ""
-        while True:
-            objects = str(await self.wss.recv()).split("")
+        final = False
+        while not final:
+            objects = str(await self.wss.recv()).split(DELIMITER)
             for obj in objects:
                 if obj is None or obj == "":
                     continue
                 response = json.loads(obj)
                 if response.get("type") == 1:
-                    cache_len = len(stream_cache)
-                    adaptive_card = response["arguments"][0]["messages"][0]["adaptiveCards"][0]["body"][0]["text"]
-                    stream_cache = adaptive_card
-                    adaptive_card = adaptive_card[cache_len:]
-                    yield adaptive_card
+                    yield False, response["arguments"][0]["messages"][0]["adaptiveCards"][0]["body"][0]["text"]
```

Comment:
I want it to stream token by token

Reply:
It makes it easier than having to clear out old outputs

Reply:
That update logic is moved to main(), not removed.

Reply:
Makes sense. Ok got it

Reply:
I'm guessing microsoft/openai designed it like that for a reason, in the future the update might change text from the previous update.